Makefile

```diff
@@ -96,23 +96,23 @@ acceptance-cluster-setup-kind:
 	@./scripts/acceptance-cluster-setup-kind.sh
 
 test-acceptance: showfocus
-	ginkgo -nodes ${GINKGO_NODES} -stream -slowSpecThreshold ${GINKGO_SLOW_TRESHOLD} -randomizeAllSpecs --flakeAttempts=${FLAKE_ATTEMPTS} -failOnPending acceptance/. acceptance/api/v1/. acceptance/apps/.
+	ginkgo -v -nodes ${GINKGO_NODES} -stream -slowSpecThreshold ${GINKGO_SLOW_TRESHOLD} -randomizeAllSpecs --flakeAttempts=${FLAKE_ATTEMPTS} -failOnPending acceptance/. acceptance/api/v1/. acceptance/apps/.
```

Comment:
enabling verbose mode in CI will make logs longer and sometimes GA UI has trouble fetching the logs, especially when they are long. It should be ok to revert to non-verbose.

Reply:
Ok. Will do. Already in a separate commit, so it will be simple to back out.

Reply:
Done. Rebase omitting the relevant commit